pkg/collector/corechecks/oracle/common/utils.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package common
+
+import (
+	"hash/fnv"
+	"strconv"
+)
+
+func GetQuerySignature(statement string) string {
+	h := fnv.New64a()
+	h.Write([]byte(statement))
+	return strconv.FormatUint(h.Sum64(), 10)
+}
```

Comment:
I don't think this is functionally equivalent to the obfuscation code, which uses a `mmh3` algorithm:

```python
def compute_sql_signature(normalized_query):
    """
    Given an already obfuscated & normalized SQL query, generate its 64-bit hex signature.
    """
    if not normalized_query:
        return None
    # Note: please be cautious when changing this function as some features rely on this
    # hash matching the APM resource hash generated on our backend.
    return format(mmh3.hash64(ensure_bytes(normalized_query), signed=False)[0], 'x')
```

It's important that this matches how APM calculates the `resource_hash` for sql resources in order for us to do linking in our backend between APM and DBM data. In order to do this, you can copy [the code here](https://github.com/DataDog/dd-go/blob/prod/trace/model/resource.go#L192-L195).

```go
func HashResourceMurmur3(r string) string {
	return strconv.FormatUint(murmur3.Sum64([]byte(r)), 16)
}
```

Reply:
thx. that's tracked here: datadoghq.atlassian.net/browse/DBM-2292